CHANGELOG.md

```diff
@@ -4,10 +4,14 @@
 
 ### Changed
 
+- [BREAKING] `Avatar`: added `creatable` prop with default value `false`. ([@driesd](https://github.com/driesd) in [#735](https://github.com/teamleadercrm/ui/pull/735))
```

Comment:
Is this really a breaking change?

Reply:
I would say the `creatable` prop itself is not a breaking change. But what about the fact that `Avatar` now has a anon user default, is that breaking or just an added feature?

Reply:
The `creatable` prop addition is not breaking at all, it was a copy/paste mistake 🙈

Reply:
Then it's time to remove it :-D